std/traits.d

```diff
@@ -3853,6 +3853,7 @@ package alias Identity(alias A) = A;
 /**
    Yields `true` if and only if `T` is an aggregate that defines
    a symbol called `name`.
+   $(NOTE Use `__traits(hasMember, T, name)` instead in new code.)
```

Comment:
Well, I find this situation a bit comical. Why do we even keep the abstraction if we recommend using `__traits`? From my perspective, we either deprecate the abstraction and justify this by the fact that you can use traits directly, or we leave it in the name of "__traits is ugly, users should use the stdlib abstractions".